pages/index.js

```diff
@@ -103,8 +106,20 @@ export default function Home({ data }) {
             height="638"
           />
         </div>
+        <a
+          href="https://github.com/EddieHubCommunity/LinkFree/discussions"
+          legacyBehavior
```

Comment:
Looks good, maybe we need it to open in a new tab?

Reply:
Correct @eddiejaoude I did that please take a look